.dev/compare_branches.R

```diff
@@ -346,10 +346,18 @@ if (length(packages) > 50L) {
   )
 }
 
+df_otherwise <- tibble::tibble(
```

Comment:
could you use tidyverse style here?

https://style.tidyverse.org/syntax.html#long-lines

Reply:
sorry. see https://github.com/r-lib/lintr/pull/982/commits/1798de0810868a5c7c09c5e84d2962661dc022c9

Reply:
np. understand styler::style is the easiest approach for such a request. and we should probably comply with that globally anyway.